mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+            pass
+        outputFileClimo = '{}/mocStreamfunction_years{:04d}-{:04d}.nc'.format(
+                           outputDirectory, startYearClimo, endYearClimo)
+
+        # Compute and plot annual climatology of MOC streamfunction
+        print '\n  Compute and/or plot post-processed MOC climatological '\
+              'streamfunction...'
+        if not os.path.exists(outputFileClimo):
+            print '   Load data...'
+            ds = open_multifile_dataset(
+                fileNames=inputFilesClimo,
+                calendar=calendar,
+                simulationStartTime=simulationStartTime,
+                timeVariableName='Time',
+                variableList=variableList,
+                variableMap=variableMap,
+                startDate=startDateClimo,
+                endDate=endDateClimo)
+
+            # Compute annual climatology
+            annualClimatology = climatology.compute_annual_climatology(
+                                ds, calendar)
+
+            horizontalVel = annualClimatology.avgNormalVelocity
+            verticalVel = annualClimatology.avgVertVelocityTop
+            # Convert to numpy
+            horizontalVel = horizontalVel.values
+            verticalVel = verticalVel.values
+            velArea = verticalVel * areaCell[:, np.newaxis]
+
+            print '   Compute global MOC...'
+            mocLatGlobal = np.arange(-90.0, 90.1, 1.)  # 1deg meridional bins
+            mocTop = np.zeros([np.size(mocLatGlobal), nVertLevels+1])
+            for iLat in range(1, np.size(mocLatGlobal)):
+                indlat = np.logical_and(latCell >= mocLatGlobal[iLat-1],
+                                        latCell < mocLatGlobal[iLat])
+                mocTop[iLat, :] = mocTop[iLat-1, :] + \
+                                  velArea[indlat, :].sum(axis=0)
+            # convert m^3/s to Sverdrup
+            mocGlobal = mocTop * m3ps_to_Sv
+            mocGlobal = mocGlobal.T
+
+            print '   Compute regional MOC...'
+            print '     Compute volume transport through southern transect...'
+            transportZEdge = np.zeros([nVertLevels, maxEdgesInTransect])
+            for i in range(maxEdgesInTransect):
+                if transectEdgeGlobalIDs[i] == 0:
+                    break
+                # subtract 1 because of python 0-indexing
+                iEdge = transectEdgeGlobalIDs[i] - 1
+                transportZEdge[:, i] = horizontalVel[iEdge, :] * \
+                    transectEdgeMaskSigns[iEdge, np.newaxis] * \
+                    dvEdge[iEdge, np.newaxis] * \
+                    refLayerThickness[np.newaxis, :]
+            transportZ = transportZEdge.sum(axis=1)
+
+            print '     Compute regional MOC...'
+            indRegion = np.where(regionCellMask == 1)
+            mocLatRegional = latCell[indRegion]
+            mocLatRegional = np.arange(np.amin(mocLatRegional)-0.5,
+                                       np.amax(mocLatRegional)+0.5, 0.5)
```

Comment:
I agree that both should be in the config file if possible, at least the resolution if not the bounds.